### ISSUE TITLE
Tooltip parameter improvement

### ISSUE THREAD

**Issue by valadem** [terrytangyuan/autoplotly#5]
Hi Terry,

In my autoplotly visualizations (pca in my case) I would like to add information to show in the tooltip from columns of the dataframe used.

The only column that can be added in the 'tooltip' parameter is the column used in the 'colour' parameter.

There is a way to add other information in the 'tooltip' parameter than 'x', 'y', 'colour'?
If not, is this a feature that will be present in a future version?

**Comment by ShinyFabio** [comment]
I'm also interested. With `ggplot()` and `ggplotly()` I can do that using the label parameter like in this example:

```
plot = ggplot(data, aes_string(label = "...")) + #instead of ... put the column name of the variable that 
                                                 #you want to pass in the tooltip
              geom_point(x, y, color)

ggplotly(plot, tooltip = c(x, y, color, label)) #here put all used variable plus label
```

But it doesn't seem to work with autoplot.

**Comment by terrytangyuan** [comment]
I agree that this is a good use case. You are welcome to submit a PR to support this.
